apis/python/src/tiledb/vector_search/index.py

```diff
@@ -800,6 +800,10 @@ def _set_has_updates(self, has_updates: bool = True):
             self.group = tiledb.Group(self.uri, "r", ctx=tiledb.Ctx(self.config))
 
     def _consolidate_update_fragments(self):
+        # Disable update fragment consolidation for TileDB Cloud URIs
+        # as this is not supported.
+        if self.uri.startswith("tiledb://"):
```

Comment:
IMO we should log a message here (once only) at a low threshold (debug level).

Reply:
Added a warning for this